specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5658,6 +5658,13 @@
       "type": "string",
```

Comment:
Should this previous parameter be removed?

Reply:
i don't think so. Previous parameter is still being referenced by other operations in the spec. We only make one exception for validateMove api. Rest should stay the same.